pkg/testgridanalysis/testgridconversion/ocp_synthetic_tests.go

```diff
@@ -32,10 +32,10 @@ func (openshiftSyntheticManager) CreateSyntheticTests(rawJobResults testgridanal
 	}
 
 	for jobName, jobResults := range rawJobResults.JobResults {
-		numRunsWithoutSetup := 0
+		numRunsWithoutInstall := 0
```

Comment:
numsWithoutInstallIndicator?

Reply:
Sorry, just saw this. Still want this changed?